jsonnet/rules.libsonnet

```
@@ -537,6 +537,14 @@ function(params) {
           expr: 'sum by (namespace) (max without(instance) (container_memory_working_set_bytes{namespace=~"openshift-monitoring|openshift-user-workload-monitoring", container=""}))',
           record: 'monitoring:container_memory_working_set_bytes:sum',
         },
+        {
+          expr: 'topk(3, sum by(namespace, job)(sum_over_time(scrape_series_added[1h])))',
+          record: 'namespace_job:scrape_series_added:topk3',
+        },
+        {
+          expr: 'topk(3, sum by(namespace, job)(sum_over_time(scrape_samples_scraped[1h])))',
```

After discussing with @dgrisonnet it appears that the most interesting information to collect from `scrape_samples_scraped` would be the max number of samples collected by one target per namespace/job pair. It could be useful to see the trend of samples number over releases for instance.
In this perspective, there's no need to sum values over time (unlike for `scrape_series_added`) because it would skew the results depending on how many targets exist for a given namespace/job (e.g. there's always 1 target for all kube-state-metrics samples while the sum of samples coming from kubelet is a factor of the number of nodes). We suggest this query instead:

```suggestion
          # For each (namespace,job) pair, return the target exposing the highest number of samples then filter the top 3 items.
          expr: 'topk(3, max by(namespace,job) (topk by(namespace,job) (1, scrape_samples_scraped)))',
```